protocol/deputy-pause-module.md

```diff
@@ -0,0 +1,80 @@
+# Deputy Pause Module
+
+## Context
+
+The Foundation Safe is given the ability to act as the Security Council Safe for a limited number
+of safety-net actions on OP Mainnet. For instance, the Foundation Safe can act as the Security
+Council Safe to trigger the Superchain-wide pause function. These powers are given to the
+Foundation Safe through the `DeputyGuardianModule` installed in the Security Council Safe.
+
+## Problem Statement
+
+The Foundation Safe pre-signs certain transactions to reduce response time in case of an emergency.
+Because pre-signed Safe transactions are dependent on the nonces of that Safe, these pre-signed
+transactions become invalidated any time the Foundation Safe is required to sign literally any
+other transaction. In an attempt to reduce the overhead from this state of affairs, a *second*
+Foundation Safe sometimes referred to as the "Foundation Operations Safe" with the exact same
+configuration as the original Foundation Safe was made to be the Deputy Guardian.
+
+Even with this second Foundation Safe, pre-signed pauses are invalidated on a regular basis
+whenever an upgrade touches the `DeputyGuardianModule`. Gripes with the pre-signed pause system
+could fill a whole role of toilet paper and are not just limited to the issues noted above.
+
+## Proposed Solution
+
+We propose eliminating the pre-signed pause system entirely by introducing a `DeputyPauseModule` to
+be installed into the primary Foundation Safe. The `DeputyPauseModule` would allow a deputy account
```

Comment:
**"deputy account"** Shall we specify if this is EOA or Smart Contract?

Reply:
Let's also specify if the module will allow exactly one deputy pauser, or a mapping of various allowed pausers